gr-azure-software-radio/README.md

```diff
@@ -0,0 +1,18 @@
+# Testing
```

Comment:
As mentioned in other comments I think we merge the build and test info in from the top level repo README and also move in info from ./docs/README.md into this file